### ISSUE TITLE
queer_hack link goes to some Beijing hotel's website now

### ISSUE THREAD

**Issue by eric-unc** [unc-csxl/csxl.unc.edu#503]
Sadly they lost the URL. Unfortunately the club is dead afaik, haven't had any events since late 2022 (at least announced in their GroupMe) and I don't think the previous officers recruited anyone else to lead.

**Comment by ajaygandecha** [comment]
Good catch @eric-unc! The website link as been erased - will loop @KrisJordan in to see if the organization should be removed from the page.
